.circleci/config.yml

```diff
@@ -14,63 +15,134 @@ jobs:
       - TOOL_NODE_FLAGS: "--max-old-space-size=4096"
 
     steps:
-      - setup_remote_docker
       - checkout
 
-      # install OS dependencies
       - restore_cache:
           name: Restoring Meteor cache
-          key: meteor
+          key: meteor-{{ .Branch }}
 
-      - run: .circleci/install.sh
+      # Install Meteor
+      - run: .circleci/install_meteor.sh
 
-      - save_cache:
-          name: Saving Meteor to cache
-          key: meteor
-          paths:
-            - ~/.meteor
+      # install OS dependencies
+      - run: sudo apt-get update
+      - run: sudo apt-get install -y locales
 
-      # install app dependencies
+      # fix Meteor/Mongo locale issue on Debian
+      # https://github.com/meteor/meteor/issues/4019
+      - run: sudo locale-gen en_US.UTF-8
+      - run: sudo localedef -i en_GB -f UTF-8 en_US.UTF-8
 
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}-{{ .Branch }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-{{ .Branch }}
+
+      # Install dependencies
       - run: meteor npm install
 
-      # run tests
       - restore_cache:
           name: Restoring Meteor dev_bundle cache
-          key: dev_bundle
+          key: dev_bundle-{{ .Branch }}
+
+      # install Reaction CLI
+      - run: sudo npm install -g reaction-cli
```

Comment:
This was done previously with `yarn`. I switched to `npm` here (for consistency), but I'll like to know if there's reason why yarn is preferred.